examples/python/pre-filter/stack.yml

```diff
@@ -0,0 +1,19 @@
+version: 1.0
+provider:
+  name: openfaas
+  gateway: https://VEBA_FQDN_OR_IP
+functions:
+  pre-filter:
+    lang: python3-flask
+    handler: ./pre-filter
+    image: vmware/veba-python-pre-filter:latest
+    environment:
+      write_debug: true
+      read_debug: true
+      insecure_ssl: true
+      call_function: veba-echo
+      filter_vm: .*
```

Comment:
Does your script also work when using `'.*'` for the `filter_vm` spec? Reason I ask is because someone might accidentally set the filter to something like `*` which is invalid yaml (https://cloudslang-docs.readthedocs.io/en/v1.0/yaml_overview.html#scalars-strings-numbers-values).

Reply:
Yes, it does. I removed the quotes for the example pattern as I figured they could cause confusion - particularly if a pattern required quotes for some reason. That said `*` is also an invalid regex. However `[a-z]` is a valid regex but not valid yaml. I've just tried both - in the case of invalid yaml `faas-cli` won't let you deploy the function. However I found I wasn't catching invalid regex patterns in the function so I'll add handling for that case to a new commit. For the example you've highlighted here, I could add quotes to show that's an option, or leave them out and add a bit to the `README.md` explaining this.